README.md

```diff
@@ -1,8 +1,131 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# JJ Falcon . Jarvis Consulting
 
+
+ I completed my bachelor's degree in Computer Science at Ryerson University with a software engineering concentration in December 2020. 
+
+ The concentration allowed me to focus my academics and excelled on the entire Software Development cycle.  It also exposed me to different Data Management Methodologies. 
+
+ I am a quick learner, creative and analytical thinker who believes that in most cases, there is no need to reinvent the wheel but look for an opportunity to make existing projects better and more efficient. 
+
+ I have a customer-centric approach to my work ethics.   I am a firm believer that success at work means helping others be better at what they do.  There is no greater motivation for me than to see my work or be a part of a team whose work has a direct impact on making other people's life easier.  This is what excites me about the software industry. 
+
+ Throughout my coop experiences, I have gained a great understanding and hands-on experience of the software development life cycle.  This includes business requirements gathering,  designing solutions,  implementing and maintaining the solution in order to produce results that are meaningful to the organization. 
+
+ Additionally, through the Jarvis training program, I have had the opportunity to use some of the technologies in manipulating and mining data. 
+
+ I also have a passion for learning new things, and I continually challenge myself to find every opportunity to learn or be a part of something new and innovative. 
+
+ I understand the importance and value of data to help companies make better business decisions and be competitive in the market, and I am looking for opportunities in this field.
```

Comment:
could you please condense it into a single paragraph? (make it shorter too) multi-paragraph doesn't look good on our app

Reply:
e.g. https://github.com/jarviscanada/jarvis_data_eng_mathew